src/mprisServer.c

```diff
@@ -17,6 +18,15 @@
 #define PROPERTIES_INTERFACE "org.freedesktop.DBus.Properties"
 #define CURRENT_TRACK -1
 
+typedef GVariant* (*ProduceVariantCb)(const char *valueStr);
+
+typedef struct {
```

Comment:
Let's avoid the typedef for the struct. That way we stay consistent with the other struct (MprisData) which also avoids it.

Reply:
Fixed